src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/models/AmbiguousRelationBetweenTables.kt

```diff
@@ -0,0 +1,7 @@
+package org.vaccineimpact.orderlyweb.models
```

Comment:
Shouldn't this be in db rather than models? Appreciate that the two are related, but since this is about tables..

Reply:
Yes you're totally right, no idea why I put them there